### PR TITLE
[zh] translate zh/docs/reference/config-api/client-authentication.v1beta1

### PR DESCRIPTION
**Pull request by howieyuen**
since client-authentication.v1beta1 and client-authentication.v1 are basically the same, so I put these two docs in one PR.

### CODE REVIEW

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md [hunk 1]
```diff
@@ -60,95 +77,135 @@ should use to contact the API.</td>
...
+<!--
 Cluster contains information to allow an exec plugin to communicate
 with the kubernetes cluster being authenticated to.
 
 To ensure that this struct contains everything someone would need to communicate
 with a kubernetes cluster (just like they would via a kubeconfig), the fields
 should shadow "k8s.io/client-go/tools/clientcmd/api/v1".Cluster, with the exception
 of CertificateAuthority, since CA data will always be passed to the plugin as bytes.
+-->
+Cluster 中包含允许 exec 插件与 Kubernetes 集群进行通信身份认证时所需
```
Comment (ydFu) [inline content/zh/docs/reference/config-api/client-authentication.v1beta1.md:95]:
```suggestion
Cluster 中包含允许 exec 插件与 Kubernetes 集群进行通信身份认证时所需
```
Referring to other Chinese localizations, the proper noun `cluster` is usually translated

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md [hunk 2]
```diff
@@ -60,95 +77,135 @@ should use to contact the API.</td>
...
+的信息。
+
+为了确保该结构体包含需要与 Kubernetes 集群进行通信的所有内容（就像通过 Kubeconfig 一样），
+该字段应该映射到 "k8s.io/client-go/tools/clientcmd/api/v1".cluster，
```
Comment (ydFu) [inline content/zh/docs/reference/config-api/client-authentication.v1beta1.md:99]:
```suggestion
该字段应该映射到 "k8s.io/client-go/tools/clientcmd/api/v1".cluster，
```
Referring to other Chinese localizations, the proper noun `cluster` is usually translated

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md [hunk 3]
```diff
@@ -60,95 +77,135 @@ should use to contact the API.</td>
...
+<!-- 
 In some environments, the user config may be exactly the same across many clusters
 (i.e. call this exec plugin) minus some details that are specific to each cluster
 such as the audience.  This field allows the per cluster config to be directly
 specified with the cluster info.  Using this field to store secret data is not
 recommended as one of the prime benefits of exec plugins is that no secrets need
-to be stored directly in the kubeconfig.</td>
+to be stored directly in the kubeconfig. 
+-->
+<p>在某些环境中，用户配置可能对很多集群而言都完全一样（即调用同一个 exec 插件），
```
Comment (ydFu) [inline content/zh/docs/reference/config-api/client-authentication.v1beta1.md:203]:
```suggestion
<p>
在某些环境中，用户配置可能对很多集群而言都完全一样（即调用同一个 exec 插件），
```

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md [hunk 4]
```diff
@@ -180,10 +241,16 @@ the transport.
...
 <td>
+<!-- 
    Cluster contains information to allow an exec plugin to communicate with the
 kubernetes cluster being authenticated to. Note that Cluster is non-nil only
 when provideClusterInfo is set to true in the exec provider config (i.e.,
-ExecConfig.ProvideClusterInfo).</td>
+ExecConfig.ProvideClusterInfo).
+-->
+   此字段中包含的信息使得 exec 插件能够与要访问的 Kubernetes 集群通信。
+   注意，cluster 字段只有在 exec 驱动的配置中 provideClusterInfo
```
Comment (ydFu) [inline content/zh/docs/reference/config-api/client-authentication.v1beta1.md:251]:
```suggestion
   注意，cluster 字段只有在 exec 驱动的配置中 provideClusterInfo
```
Referring to other Chinese localizations, the proper noun `cluster` is usually translated

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md (hunk 1 above)
Comment (howieyuen) [reply content/zh/docs/reference/config-api/client-authentication.v1beta1.md:95]: the Cluster is a Go Struct，so I leave it a lone

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md (hunk 2 above)
Comment (howieyuen) [reply content/zh/docs/reference/config-api/client-authentication.v1beta1.md:99]: the Cluster is a Go Struct，in package `k8s.io/client-go/tools/clientcmd/api/v1`

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md (hunk 4 above)
Comment (howieyuen) [reply content/zh/docs/reference/config-api/client-authentication.v1beta1.md:251]: the cluster is a field name of a struct

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md (hunk 1 above)
Comment (ydFu) [reply content/zh/docs/reference/config-api/client-authentication.v1beta1.md:95]: thank,I get it

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md (hunk 2 above)
Comment (ydFu) [reply content/zh/docs/reference/config-api/client-authentication.v1beta1.md:99]: thank,I get it

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md (hunk 4 above)
Comment (ydFu) [reply content/zh/docs/reference/config-api/client-authentication.v1beta1.md:251]: thank,I get it

File: content/zh/docs/reference/config-api/client-authentication.v1beta1.md (hunk 3 above)
Comment (howieyuen) [reply content/zh/docs/reference/config-api/client-authentication.v1beta1.md:203]: thanks, I lost the end tag

**Pull request merged**